pdsfile/pds3file/__init__.py

```diff
@@ -0,0 +1,139 @@
+################################################################################
+# pds3file subpackage & Pds3File subclass with PdsFile as the parent class
+################################################################################
+
+from pdsfile import *
+from . import rules as pdsfile_rules
```

Comment:
Is there a reason to rename rules to pdsfile_rules?

Reply:
Fixed